### DISCUSSION TITLE
Update the Security Finding class by changing the activity_id enum label

### DISCUSSION THREAD

**Discussion by pagbabian-splunk** [ocsf/ocsf-schema#492] [Proposals]
The Security Finding class has the following enum:
`0 - Unknown`
`1 - Generate`
`2 - Update`
`99 - Other`

Generate appears to be equivalent to Create which is used frequently throughout the schema.

This proposal changes Generate to Create.

Note, it is technically NOT a breaking change, however by convention the optional value populated in `activity_name` would change from "Generate" to "Create"

**Comment by irakledibm** [comment]
"Create" makes sense to me. I would probably add "Close" as well.
